src/main/java/net/dv8tion/jda/api/entities/PermissionOverride.java

```diff
@@ -100,6 +101,63 @@
     @Nonnull
     JDA getJDA();
 
+    /**
+     * This method will return the {@link net.dv8tion.jda.api.entities.IPermissionHolder PermissionHolder} of this {@link net.dv8tion.jda.api.entities.PermissionOverride PermissionOverride}.
+     * It can be used to e. g. get the general permissions of that PermissionHolder no matter if it is a {@link net.dv8tion.jda.api.entities.Member Member} or a {@link net.dv8tion.jda.api.entities.Role Role}.
+     *
+     * <p><b>To get the concrete Member or Role, use {@link PermissionOverride#getMember()} or {@link PermissionOverride#getRole()}!</b>
+     *
+     * <h2>Example</h2>
+     * <pre><code>
```

Comment:
Why does thie getter need a complex example?

Reply:
Does it even need a before-after comparison?
All the examples I saw in the Javadoc so far were always just what you can/should do with it, not what you can replace with it.

Reply:
To show a sample use-case and how it makes that easier. Will remove the old one and simplify it.

Reply:
I don't think the example is necessary, its a getter everyone knows how to use it.